### PR TITLE
Installation: Add Chaotic AUR and `downgrade` mention

### PR DESCRIPTION
**Pull request by Visual-Dawg**
Just sharing what I find useful.

The Chaotic Aur is awesome and downgrade makes it quick and easy to, well, downgrade, especially with the Chaotic Aur.

### CODE REVIEW

**Comment by alba4k** [comment]
> we recommend

chaotic-aur is a really nice thing to have, but I wouldn't officially recommend it as it isn't a great idea in principle. Of course it's not a complete anarchy, but I still think that sentence should be reworded sligthly

**Comment by Visual-Dawg** [comment]
True, I changed it

**Review by alba4k — CHANGES_REQUESTED** [review]
pre-built not pre-build

also, the part after the comma doesn't really make sense grammatically


"and be able to perform easy downgrades", maybe?

**Comment by alba4k** [comment]
It should probably also be made clear that this is all regarding Arch Linux, as NixOS is also officially supported

**Comment by Visual-Dawg** [comment]
This is within the Arch Linux install card, so it should be clear enough

**Comment by alba4k** [comment]
You're right, my bad ahah

File: pages/Getting Started/Installation.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ hyprland-git (AUR) - compiles from latest source
 hyprland - binary x86 tagged release
 ```
 
+If you decide to use the `git` version, you can use the [Chaotic Aur](https://aur.chaotic.cx/) to get pre-built binaries.
```
Comment (alba4k) [inline pages/Getting Started/Installation.md:45]: maybe change this part to

> the `-git` version from the AUR

**Review by alba4k — CHANGES_REQUESTED** [review]

**Review by fufexan — APPROVED** [review]
LGTM

**Comment by vdawg-git** [comment]
@alba4k  Good idea, I also experienced this. 
Vax could also just bumb all package versions when there is an ABI break, but I suppose adding a warning or removing the chaotic AUR is better

**Pull request merged**